storage/netapp_e/na_eseries_storage_system.py

```diff
@@ -212,7 +224,7 @@ def main():
                 changed = True
             if array_detail['wwn'] != array_wwn and array_wwn is not None:
                 module.fail_json(
-                    msg='It seems you may have specified a bad WWN. The storage system ID you specified, %s, " + '
+                    msg='It seems you may have specified a bad WWN. The storage system ID you specified, %s, ' +
                         'currently has the WWN of %s' % (ssid, array_detail['wwn']))
```

Comment:
Ok, I'm not a python pro; thanks for the detail.  I just pushed a commit with that corrects.